File: MyTournamentApp/app/src/main/java/com/example/ourtournament/Objetos/Torneo.java
package com.example.ourtournament.Objetos;

public class Torneo
{
    public int IDTorneo;
    public String NombreTorneo;
    public String ContraseniaDeAdministrador;
    public String LinkParaUnirse;

    public Torneo(int idtorneo, String nombreTorneo, String contraseniaDeAdministrador, String linkParaUnirse)
    {
        IDTorneo = idtorneo;
        NombreTorneo= nombreTorneo;
        ContraseniaDeAdministrador = contraseniaDeAdministrador;
        LinkParaUnirse = linkParaUnirse;
    }

    public Torneo()
    {
        this(0,"","","");
    }
}

File: MyTournamentAPI/OurTournamentAPI/Models/Torneo.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;

namespace OurTournamentAPI.Models
{
    public class Torneo
    {
        private int _IDTorneo;
        private String _nombreTorneo;
        private String _contraseniaDeAdministrador;
        private String _linkParaUnirse;

        public int IDTorneo { get => _IDTorneo; set => _IDTorneo = value; }
        public String NombreTorneo { get => _nombreTorneo; set => _nombreTorneo = value; }
        public String ContraseniaDeAdministrador { get => _contraseniaDeAdministrador; set => _contraseniaDeAdministrador = value; }
        public String LinkParaUnirse { get => _linkParaUnirse; set => _linkParaUnirse = value; }

        public Torneo(int idtorneo, string nombreTorneo, string contraseniaDeAdministrador, string linkParaUnirse)
        {
            _IDTorneo = idtorneo;
            _nombreTorneo= nombreTorneo;
            _contraseniaDeAdministrador = contraseniaDeAdministrador;
           _linkParaUnirse = linkParaUnirse;
        }

        public Torneo() : this(0, "", "", "")
        {
           
        }



    }
}